website/source/guides/advanced-scheduling/spread.html.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: "guides"
+page_title: "Spread"
+sidebar_current: "guides-advanced-scheduling"
+description: |-
+    The spread stanza allows operators to increase failure tolerance in their Nomad clusters by allowing them to distribute their workloads in a customized way based on attributes or client metadata.
+---
+
+# Increasing Failure Tolerance with Spread
+
+The Nomad scheduler uses a bin packing algorithm when making job placements on
+nodes to optimize resource utilization and density of applications. Although
+this feature ensures that cluster resources are being used efficiently, it does
+not necessarily promote maximum failure tolerance of jobs across nodes.
+
```

Comment:
Wondering if we could include a diagram here to show a hypothetical distribution of allocs with binpacking. Can take something from our Hashiconf talk and modify it to use here..